seqr/views/utils/dataset_utils.py

```diff
@@ -413,13 +404,10 @@ def _load_rna_seq(model_cls, file_path, user, mapping_file, ignore_extra_samples
         sample_project_tuples=samples_by_id.keys(),
         data_source=data_source,
         sample_id_to_individual_id_mapping=sample_id_to_individual_id_mapping,
-        raise_unmatched_error_template=None if ignore_extra_samples else 'Unable to find matches for the following samples: {sample_ids}'
+        raise_unmatched_error_template=None if ignore_extra_samples else 'Unable to find matches for the following samples: {sample_ids}',
+        sample_id_to_tissue_type=sample_id_to_tissue_type,
     )
 
-    warnings = []
-    if validate_samples:
-        samples = validate_samples(samples, sample_id_to_tissue_type, warnings)
-
     # Delete old data
     to_delete = model_cls.objects.filter(sample__in=samples).exclude(sample__data_source=data_source)
     prev_loaded_individual_ids = set(to_delete.values_list('sample__individual_id', flat=True))
```

Comment:
It is still a problem for the `data_source` attribute of the sample. We don't update the attribute for the existing sample.

Reply:
I don't 100% follow - are you saying that if we had previously loaded data for a specific tissue type we are not updating the data source when we reload it? Because if so that seems like a pretty straightforward change to make

Reply:
Suppose we have previously loaded samples from a data source in the DB when we reload data for the samples from a different data source. In that case, we delete the existing data from the previous data source before loading the new data. But we don't update the data source of the sample model to the latest data source. Do you think it is an issue?

Reply:
yes I think we should do a bulk update of the sample models to the latest data source